lua/gt.lua

```diff
@@ -9,6 +9,11 @@ return function (net_conf, numa_table)
 	-- Change these parameters to configure the Grantor.
 	gt_conf.ggu_src_port = 0xA0A0
 	gt_conf.ggu_dst_port = 0xB0B0
+	gt_conf.frag_scan_timeout_ms = 100;
```

Comment:
Let's suggest
```
-- Scan the whole fragment table in 2 minutes.
gt_conf.frag_scan_timeout_ms = math.floor(2*60*1000 / gt_conf.frag_bucket_num + 0.5)
```
Due to the dependency on variable `gt_conf.frag_bucket_num`, this initialization must come later in the code.